client/src/main/java/io/pravega/client/stream/SegmentReaderSnapshot.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.pravega.client.stream;
+
+import io.pravega.client.segment.impl.Segment;
+
+/**
+ * SegmentReaderInfo provides the information about the current position, segment id and the status of segment reader.
```

Comment:
I guess this should be 'SegmentReaderSnapshot'

Reply:
Done